0.19/Dockerfile

```diff
@@ -1,20 +1,20 @@
-FROM debian:stable-slim
+ARG ARCH=amd64
+FROM debian:stable-slim as build
 
 LABEL maintainer.0="João Fonseca (@joaopaulofonseca)" \
   maintainer.1="Pedro Branco (@pedrobranco)" \
   maintainer.2="Rui Marinho (@ruimarinho)"
 
 RUN useradd -r bitcoin \
   && apt-get update -y \
-  && apt-get install -y curl gnupg gosu \
```

Comment:
Care to explain why you would remove gosu? Maybe better to open an issue and open a separate PR for this mod